packages/forms-web-app/src/dynamic-forms/route.js

```diff
@@ -27,12 +27,20 @@ router.post(
 router.post('/questionnaire/:referenceId/', getJourneyResponse(), validationErrorHandler, submit);
 
 router.get(
```

Comment:
oh! and if you did make that dictionary I mentioned then you could loop through its keys to make these routes!